src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2815,6 +2815,7 @@ because xbuild doesn't support framework reference assemblies.
 	<ItemGroup>
 		<AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)\lib\*\libsqlite3_xamarin.so" Condition="$(_Assemblies.Contains('Mono.Data.Sqlite.dll'))" />
 		<AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)\lib\*\libMonoPosixHelper.so" Condition="$(_Assemblies.Contains('Mono.Posix.dll'))" />
+		<AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)\lib\*\libmono-native.so" Condition="$(_Assemblies.Contains('mscorlib.dll'))" />
```

Comment:
I think the `Condition` can be skipped here; I don't think it's *possible* to write a Xamarin.Android app which doesn't reference `mscorlib.dll`.